### DISCUSSION TITLE
Welcome to BaldPhone Discussions!

### DISCUSSION THREAD

**Discussion by UriahShaulMandel** [UriahShaulMandel/BaldPhone#793] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sami155** [comment]
Well i have a question:
i had forked this repo but i translated the strings.xml on this repo (not on my fork). but after submitting it, i saw, that some bot gave me an error or so. and after that i deleted my fork (idk why) and now im unable to find my translated string.xml file.

Do you know if it got deleted by a bot, or did you recivie my submission?

Thanks

**Comment by outerowl** [comment]
> ## 👋 Welcome!
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
>   * Ask questions you’re wondering about.
>   * Share ideas.
>   * Engage with other community members.
>   * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 

Such a great app very helpful for my grandfather why are you not improving this or working on this to make sure it's get updated with latest android versions.
